README.md

```diff
@@ -1,6 +1,23 @@
 # Welcome to Crowbar Angular Prototype
 [![Build Status](https://travis-ci.org/crowbar/crowbar-ui.svg?branch=master)](https://travis-ci.org/crowbar/crowbar-ui)
 
+### Setting up a development environment with Docker
+
+1. make sure `docker` is installed and the `docker` daemon is running.
+2. build the image from the [Dockerfile](Dockerfile):
+```
+docker build -t crowbar-ui .
+```
+3. run the container:
+```
+docker run -d --name "crowbar-ui_container" -v $(pwd)/assets:/crowbar-ui/assets -p 3000:3000 crowbar-ui
+```
+4. run the `gulp` watchdog:
+```
+docker exec -ti crowbar-ui_container gulp
+```
+5. work on the assets folder in your local git clone and the changes will be immediately visible in the browser.
```

Comment:
re: unit tests

They can be run on teh container and if we use supervisord for them, we can make them run on every change with the `--watch` option. And if we mount the current dir instead of the assets folder, we can have the coverage report available on the host :D